aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -408,6 +411,21 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         }
     }
 
+    fun isMediaModeEnabled(): Boolean {
```

Comment:
Will this method be used in a client app (e.g. WordPress) in the future?  I'm just wondering since it isn't used in the library or demonstration app.